# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libx11/x11.pc.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./x11.pc.in.orig	2008-02-28 16:30:50.000000000 +0000
+++ ./x11.pc.in	2008-02-28 16:31:25.000000000 +0000
@@ -8,8 +8,8 @@
 Name: X11
 Description: X Library
 Version: @PACKAGE_VERSION@
-Requires: xproto @XKBPROTO_REQUIRES@
-Requires.private: xau xdmcp @X11_EXTRA_DEPS@
+Requires: xproto @XKBPROTO_REQUIRES@ @X11_EXTRA_DEPS@
+Requires.private: xau xdmcp
 Cflags: -I${includedir} @XTHREAD_CFLAGS@
 Libs: -L${libdir} -lX11
 Libs.private: @XTHREADLIB@
